client/src/backoffice/EventsPage.tsx

```diff
@@ -303,10 +303,16 @@ const EventField: React.FC<EventFieldProps> = ({ title, name, disabled, type })
   );
 };
 
-const EventList: React.FC = () => {
-  const [events, fetchingEvents, fetchEventsError] = useAsync(getEvents);
+export const EventList: React.FC = () => {
   const [criteria, setCriteria] = useState<string>('');
 
+  const users = authClient.user['https://poap.xyz/roles'];
+  const isAdmin = Array.isArray(users) ? users.includes('administrator') : false;
```

Comment:
We should use the `ROLES` constant instead of `'administrator'` string: [line 1]( https://github.com/Xivis/poap/blob/ADMINS/qr-page/client/src/lib/constants.ts#L1)
We can easily make mistakes on strings and are hard to refactor afterwards if backend decides to change this.

Code example:
```typescript
const isAdmin = Array.isArray(users) ? users.includes(ROLES.administrator) : false;
```